tiatoolbox/annotation/storage.py

```diff
@@ -1170,12 +1238,12 @@ def write_geojson_to_file_handle(file_handle: IO):
             file_handle.write("]}")
 
         return self._dump_cases(
-            fp=fp,
+            full_path=full_path,
```

Comment:
Again full path doesn't make sense as a name here. It can be an IO object or a file path (full or relative etc.).

Reply:
i've changed this back to fp again